README.md

```diff
@@ -1,55 +1,169 @@
+
+You can add "Long Answer Recap" components to mentoring blocks later on
+in the course to provide a read-only view of any answer that the student
+entered earlier.
+
+The read-only answer is rendered as a quote in the LMS:
+
+![Answer Read-Only](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/answer-3.png)
+
+### Self-assessment MCQs
+
+Multiple choice questions are represented by the `<mcq>` element. The
+`<mcq>` element supports the following attributes:
+
+* `name` - Unique name that identifies the question withing a course.
+* `weight` - The weight is used when computing total grade/score of
+  the mentoring block. The larger the weight, the more influence this
+  question will have on the grade. Value of zero means this question
+  has no influence on the grade (float, defaults to `1`).
+* `type` - Can be set to `choices` (default) or `rating`.
+* `low` - Sets the label of the lowest value. Only makes sense for
+  questions of the `rating` type (string; defaults to `"Less"`).
+* `high` - Sets the label of the lowest value. Only makes sense for
+  questions of the `rating` type (string; defaults to `"More"`).
+
+The `<mcq>` element can contain the following child elements:
+
+* `<question>` - The question text to display above the radio buttons.
+* `<choice>` - Defines a choice corresponding to a single radio
+  button.
+* `<tip>` - Defines feedback tips displayed when student submits their
+  answer.
+
+The `<choice>` elements support a single (required) attribute
+`value`. The should be a string which is unique among the choices of a
+single MCQ.
+
+The contents of the `<tip>` element specify the message to display
+next to the question when the question is submitted. It supports two
+mutually exclusive attributes `display` and `reject`. The value of
+the attributes should be a comma-separated list of choice values for
+which the tip should be shown. When using `display`, the listed
+choices are considered "correct" and if one of the listed choices is
+selected when submitting an answer, the question will be considered
+complete. The values under `reject` on the other hand contain
+questions which are not considered correct and the question will not
+be considered completed until the student submits a choice from the
+`display` list instead.
+
+#### Rating MCQ
+
+When constructing questions where the student rates some topic on the
+scale from `1` to `5`, you can set the `type` attribute of `<mcq>` to
+`rating`, which automatically generates choices with values from `"1""`
+to `"5"`. The `low` and `high` attributes specify the text shown next
+to the lowest and highest valued choice.
+
+#### Example
+
+The below example sets up two MCQs. The second one is of the `rating`
+type.
 
 ```xml
-<mentoring url_name="quizz_1" enforce_dependency="false">
-    <quizz name="quizz_1_1" type="choices">
-        <question>Do you like this quizz?</question>
+<mentoring url_name="mcq_1">
```

Comment:
This entire README probably needs another look over to eliminate XML snippets now that we don't really support creating those manually anymore.